worldengine/generation.py

```diff
@@ -200,7 +200,7 @@ def generate_world(w, step):
 
     # Prepare sufficient seeds for the different steps of the generation
     rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
-    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
+    sub_seeds = rng.randint(0, 2147483647, size=100)  # sys.maxsize didn't quite work
```

Comment:
I think uint32 is fine, numpy doesn't seem to have a problem.

```
In [15]: numpy.random.randint(0,numpy.iinfo(numpy.uint32).max)
Out[15]: 250145318
```

Numpy only cries when using uint64:
```
In [14]: numpy.random.randint(0,numpy.iinfo(numpy.uint64).max)
---------------------------------------------------------------------------
OverflowError                             Traceback (most recent call last)
<ipython-input-14-311a6f00f03d> in <module>()
----> 1 numpy.random.randint(0,numpy.iinfo(numpy.uint64).max)

/usr/lib/python2.7/dist-packages/numpy/random/mtrand.so in mtrand.RandomState.randint (numpy/random/mtrand/mtrand.c:7601)()

OverflowError: Python int too large to convert to C long
```